fix: update zh.csv

szufisher opened the pull request:
<!--

Some key notes before you open a PR:

 1. Select which branch should this PR be merged in?
 2. PR name follows [convention](http://karma-runner.github.io/4.0/dev/git-commit-msg.html)
 3. All tests pass locally, UI and Unit tests
 4. All business logic and validations must be on the server-side
 5. Update necessary Documentation
 6. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes


Also, if you're new here

- Documentation Guidelines => https://github.com/frappe/erpnext/wiki/Updating-Documentation

- Contribution Guide => https://github.com/frappe/frappe/blob/develop/.github/CONTRIBUTING.md

- Pull Request Checklist => https://github.com/frappe/erpnext/wiki/Pull-Request-Checklist

-->

> Please provide enough information so that others can review your pull request:

<!-- You can skip this if you're fixing a typo or updating existing documentation -->

> Explain the **details** for making this change. What existing problem does the pull request solve?

<!-- Example: When "Adding a function to do X", explain why it is necessary to have a way to do X. -->

> Screenshots/GIFs

<!-- Add images/recordings to better visualize the change: expected/current behviour -->
Update zh.csv

szufisher committed:
fix: update zh.csv

szufisher committed:
fix: update zh.csv

szufisher committed:
fix: make server messages translatable in boot

1. extracted server messages when boot
2. removed exclude server message from public folder

szufisher committed:
fix: make get untranslated for specified app

make get untranslated for specified app possible

ankush commented:
The diff suggests it does more than just updating zh.csv. Please share changelog and brief info about what problem this PR addresses.

szufisher committed:
Update translate.py

restore the change

szufisher committed:
Update translate.py

restore changes

szufisher commented:
> The diff suggests it does more than just updating zh.csv. Please share changelog and brief info about what problem this PR addresses.

restored the changes in translate.py, will handle the change in translate.py via another separate PR.

thanks.

szufisher committed:
Delete zh.csv

szufisher committed:
fix: update zh.csv

make sure in Notepad++ Edit ->EOL Conversion is Unix style as LF instead of windows style CR LF, this will make source text with line breaks translatable

szufisher commented:
@surajshetty3416 can you help to review, thanks.

szufisher committed:
fix: update zh.csv

surajshetty3416 commented:
https://github.com/frappe/erpnext/pull/24647#issuecomment-782086489

[More on the translation contribution process](https://docs.erpnext.com/docs/user/manual/en/translations#1-how-to-contribute-translations)

szufisher commented:
@surajshetty3416 

Would you mind to do me a favor: upload via csv in the backend into the translation portal?
Also I would like to know once uploaded in translation portal, how to push it as a PR?
thanks.
